plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -1775,14 +1775,20 @@ public void testBucketedTableDoubleFloat()
 
             assertTableIsBucketed(tableHandle, transaction, session);
 
+            float testFloatValue = 87.1f;
+            double testDoubleValue = 88.2;
+
             ImmutableMap<ColumnHandle, NullableValue> bindings = ImmutableMap.<ColumnHandle, NullableValue>builder()
-                    .put(columnHandles.get(columnIndex.get("t_float")), NullableValue.of(REAL, (long) floatToRawIntBits(87.1f)))
-                    .put(columnHandles.get(columnIndex.get("t_double")), NullableValue.of(DOUBLE, 88.2))
+                    .put(columnHandles.get(columnIndex.get("t_float")), NullableValue.of(REAL, (long) floatToRawIntBits(testFloatValue)))
+                    .put(columnHandles.get(columnIndex.get("t_double")), NullableValue.of(DOUBLE, testDoubleValue))
                     .buildOrThrow();
 
-            // floats and doubles are not supported, so we should see all splits
-            MaterializedResult result = readTable(transaction, tableHandle, columnHandles, session, TupleDomain.fromFixedValues(bindings), OptionalInt.of(32), Optional.empty());
-            assertEquals(result.getRowCount(), 100);
+            MaterializedResult result = readTable(transaction, tableHandle, columnHandles, session, TupleDomain.fromFixedValues(bindings), OptionalInt.of(1), Optional.empty());
+
+            boolean resultFound = result.getMaterializedRows().stream()
```

Comment:
nit: Can we build a `MaterializedRow` and assert it here

Reply:
Building MaterializedRow is complex since there are a lot of other fields in the result. Instead, I've made it simple by asserting directly on materialized rows